capellambse/extensions/pvmt.py

```diff
@@ -36,7 +36,7 @@ def from_model(
         cls, model: capellambse.MelodyModel, element: etree._Element
     ) -> PropertyValueProxy:
         """Create a PropertyValueProxy for an element."""
-        if not hasattr(model, "_pvext") or model._pvext is None:
+        if not hasattr(model, "pvext") or model.pvext is None:
```

Comment:
We briefly had touched upon this point before, but here it is in writing:

IMO this shouldn't be made part of the public API, instead we should improve the public API that we already have (i.e. the classes around `PropertyValueProxy`). The object behind `_pvext` has a very different API philosophy, to the point where one could even argue to get rid of it entirely and reimplement it based purely on the `capellambse.model` classes.